src/main/java/racingcar01/domain/CarMove.java

```diff
@@ -0,0 +1,20 @@
+package racingcar01.domain;
```

Comment:
전략패턴은 domain패키지보다 strategy패키지로
```suggestion
package racingcar01.strategy;
```
패키지 위치로 구조를 파악하기 쉽게 개선해보자.